AsyncDisplayKit/Layout/ASLayoutablePrivate.h

```diff
@@ -12,8 +12,29 @@
 
 @class ASLayoutSpec;
 @class ASLayoutOptions;
+@protocol ASLayoutable;
 
 @protocol ASLayoutablePrivate <NSObject>
-- (ASLayoutSpec *)finalLayoutableWithParent:(ASLayoutSpec *)parentSpec;
+
+/**
+ *  @abstract A display node cannot implement both calculateSizeThatFits: and layoutSpecThatFits:. This
```

Comment:
The comments you added are overall AWESOME!!

The first line of this one confuses me, and may be a copy/paste error.  If we intended to mention those methods, please add to the description to make it more clear why they're brought up.